actions/add-user-metadata.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
We can probably deploy these automatically using the Management API. See https://auth0.com/docs/api/management/v2/actions/patch-action.